src/extension/wps/wps-core/src/main/java/org/geoserver/wps/WPSResourceVoter.java

```diff
@@ -0,0 +1,21 @@
+/* (c) 2018 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wps;
+
+import org.geoserver.catalog.ResourceInfo;
+import org.geoserver.catalog.ServiceResourceVoter;
+
+public class WPSResourceVoter implements ServiceResourceVoter {
+
+    @Override
+    public boolean hideService(ResourceInfo resource) {
+        return true;
```

Comment:
Why so? WPS is a legitimate user of local layers, and it does so by avoiding making other requests, e.g., when the Execute looks like it's doing a WFS request, it's in fact avoiding the OGC machinery and goes directly to the WFS service implementation, without going thought the disptacher